base-node/index.d.ts

```diff
@@ -4,7 +4,7 @@ import { LoguxError, LoguxErrorOptions } from '../logux-error'
 import { Log, Action, Meta } from '../log'
 
 interface Authentificator {
-  (nodeId: string, token: string): Promise<boolean>
+  (nodeId: string, token: string, headers: object): Promise<boolean>
```

Comment:
You will hate me 😅.

Let’s add generic for headers time. Something like:

```ts
/**
 * …
 * @template H Remote headers.
class BaseName<H = { }> {
  remoteHeaders: H | {}
  auth: Authentificator<H | {}>
}
```

I use `| {}` to explicitly show that developer must be ready if the client will not send the header.

Reply:
I've added the generic types for remote headers. But I need pass it through `NodeOptions`.

Also I've added the docs for `NodeOptions` and `Authentificator` what generic type `H` is. It is repeating but I'm not sure if it will be clear for user.